packages/service/src/storybook/utils.ts

```diff
@@ -302,71 +305,158 @@ export function createTestFiles(
+            args: [
+                'disable-infobars',
+                ...(isHeadless ? ['--headless'] : []),
+            ],
+
+        },
+        'wdio-ics:options': {
+            logName: 'local-chrome',
+        },
+    }
+    const edgeCapability: WebdriverIO.Capabilities = {
+        browserName: 'MicrosoftEdge',
+        'ms:edgeOptions': {
+            args: [...(isHeadless ? ['--headless'] : [])],
+        },
+        'wdio-ics:options': {
+            logName: 'local-edge',
+        },
+    }
+    const firefoxCapability: WebdriverIO.Capabilities = {
+        browserName: 'firefox',
+        'moz:firefoxOptions': {
+            args: [...(isHeadless ? ['-headless'] : []),]
+        },
+        'wdio-ics:options': {
+            logName: 'local-firefox',
+        },
+    }
+    const safariCapability: WebdriverIO.Capabilities = {
+        browserName: 'safari',
+        'wdio-ics:options': {
+            logName: 'local-safari',
+        },
+    }
+    interface CapabilityMap {
             chrome: typeof chromeCapability;
+            edge: typeof edgeCapability;
             firefox: typeof firefoxCapability;
             safari: typeof safariCapability;
-            edge: typeof edgeCapability;
         }
-        const capabilityMap: CapabilityMap = {
-            chrome: chromeCapability,
-            firefox: firefoxCapability,
-            safari: safariCapability,
-            edge: edgeCapability,
-        }
-        const newCapabilities = browsers
-            .filter((browser:string): browser is keyof CapabilityMap => browser in capabilityMap)
-            .map((browser:string) => capabilityMap[browser as keyof CapabilityMap])
-        capabilities.length = 0
-        // Add the new capability to the capabilities array
-        capabilities.push(...newCapabilities)
-    } else {
-        log.error('The capabilities are not an array')
+    const capabilityMap: CapabilityMap = {
+        chrome: chromeCapability,
+        edge: edgeCapability,
+        firefox: firefoxCapability,
+        safari: safariCapability,
     }
+    const standardCapabilities = browsers
+        .filter((browser: string): browser is keyof CapabilityMap => browser in capabilityMap && browser.toLowerCase() !== 'chrome')
+        .map((browser: string) => capabilityMap[browser as keyof CapabilityMap])
+
+    capabilities.push(...standardCapabilities)
+
+    if (isMobileEmulation) {
+        devices.forEach((deviceName: string) => {
+            const foundDevice = deviceDescriptors.find((device: EmulatedDeviceType) => device.name === deviceName)
+            if (foundDevice) {
+                const chromeMobileCapability = createChromeCapabilityWithEmulationFunc(foundDevice, isHeadless)
+                // @ts-ignore because of "Argument of type 'Capabilities' is not assignable to parameter of type '(DesiredCapabilities | W3CCapabilities) & MultiRemoteCapabilities'."
+                capabilities.push(chromeMobileCapability)
```

Comment:
Which allows us to get rid of this
```suggestion
                capabilities.push(chromeMobileCapability)
```

Reply:
tnx, will fix that